onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -735,4 +736,247 @@
+    <string name="error_while_retrieving_data">Error while retrieving data</string>
+    <string name="trip_plan_from">From</string>
+    <string name="trip_plan_to">To</string>
+    <string name="trip_plan_when">When</string>
+    <string name="trip_plan_leaving">Leaving ▾</string>
+    <string name="trip_plan_arriving">Arriving ▾</string>
+
+    <string name="trip_plan_location_hint">Enter address or location name</string>
+
+    <string name="trip_plan_switch_view">Switch View:</string>
+    <string name="trip_plan_list_view">List View</string>
+    <string name="trip_plan_map_view">Map View</string>
+
+    <string name="task_progress_tripplanner_progress">Planning trip. Please wait&#8230;</string>
+    <string name="toast_no_server_selected_error">No region selected</string>
+    <string name="tripplanner_results_title">Planning Results</string>
+    <string name="tripplanner_error_dialog_title">Planning not possible</string>
+    <!-- Errors -->
+    <string name="tripplanner_error_not_defined">Sorry, something went wrong.  Please try again.</string>
+    <string name="tripplanner_error_request_timeout">Region is not responding, please try again later</string>
+    <string name="tripplanner_error_system">Region is offline, please try again later</string>
+    <string name="tripplanner_error_outside_bounds">Origin or destination is outside transportation network coverage</string>
+    <string name="tripplanner_error_path_not_found">Couldn\'t find a matching route between origin and destination</string>
+    <string name="tripplanner_error_no_transit_times">Transit times for selected date are not available</string>
+    <string name="tripplanner_error_bogus_parameter">Something went wrong.  Change your trip options and try again.</string>
+    <string name="tripplanner_error_geocode_from_not_found">Origin not found</string>
+    <string name="tripplanner_error_geocode_to_not_found">Destination not found</string>
+    <string name="tripplanner_error_geocode_from_to_not_found">Origin and destination not found</string>
+    <string name="tripplanner_error_too_close">Trip is too short</string>
+    <string name="tripplanner_error_location_not_accessible">Chosen location is not accessible</string>
+    <string name="tripplanner_error_geocode_from_ambiguous">Origin is uncertain</string>
+    <string name="tripplanner_error_geocode_to_ambiguous">Destination  is uncertain</string>
+    <string name="tripplanner_error_geocode_from_to_ambiguous">Origin and destination are uncertain</string>
+    <string name="tripplanner_error_triangle">Something went wrong.  Change your bike options and try again</string>
+    <!-- When routing from my location -->
+    <string name="tripplanner_current_location">Current Location</string>
+    <string name="tripplanner_error_no_location">Please enter a location for the start and end of your trip.</string>
+
+    <string name="tripplanner_reverse">Reverse</string>
+
+    <!-- Units -->
```

Comment:
For distance units, we already have:

    <string name="feet_abbreviation">ft</string>
    <string name="miles_abbreviation">mi</string>
    <string name="meters_abbreviation">m</string>
    <string name="kilometers_abbreviation">km</string>

Unless there is a reason not to, let's use these to be consistent throughout OBA.